packages/domain/lib/mappers/base_mappers.dart

```diff
@@ -0,0 +1,7 @@
+abstract class Mapper<I, Output> {
+  Output call(I params);
+}
+
+abstract class MapperTwoParams<InputParams, InputParamsTwo, Output> {
+  Output call(InputParams params, InputParamsTwo paramsTwo);
+}
```

Comment:
work with mappers as with usecase - if u need to pass several params - encapsulate them

Reply:
removed